d3a_interface/utils.py

```diff
@@ -0,0 +1,49 @@
+"""
+Copyright 2018 Grid Singularity
```

Comment:
Im not sure, does it needs to be updated as per current year?

Reply:
I can ask Ana, but I think that this should refer to the start date of the licensing.